inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -48,7 +48,8 @@ def __init__(self,
             self.domain = domain
 
     def simulate(self,
-                 simulator: Simulator = OpenFOAM(),
+                 simulator: Simulator = OpenFOAM(
+                     "windtunnel.openfoam.run_simulation"),
```

Comment:
Will the user have to provide this string when defining the scenario?

Reply:
Only if he would pass another simulator (like in the case of DualSPH and SplishSplash for FluidBlock). 

Since we only have OpenFOAM and this is not a permanent solution I went with this simple structure.